### PR TITLE
Implement setCurrentPage

### PR DESCRIPTION
**Pull request by Maxwell-Yang-PDFTron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -849,6 +850,21 @@ this._viewer.getPageCropBox(1).then((cropBox) => {
...
+##### setCurrentPage
+Set current page of the document.
+
+Parameters:
+
+Name | Type | Description
+--- | --- | ---
+pageNumber | integer | the page number for the target crop box. It is 1-indexed
+
+Return a Promise (will be rejected if provided page number is out of bound).
```
Comment (sgong-pdftron) [inline README.md:862]: i thought we agreed on resolve with boolean?

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]
.

File: README.md (hunk 1 above)
Comment (Maxwell-Yang-PDFTron) [reply README.md:862]: Yeah did not change that yet, sorry

**Review by sgong-pdftron — APPROVED** [review]

File: ios/RNTPTDocumentView.m [hunk 2]
```diff
@@ -2090,5 +2090,25 @@ -(void)selectAnnotation:(NSString *)annotationId pageNumber:(NSInteger)pageNumbe
...
 
+#pragma mark - Set Current Page
+
+- (bool)setCurrentPage:(NSInteger)pageNumber {
+    
+    NSError *error = nil;
+    
+    __block bool setResult;
+    
+    [self.pdfViewCtrl DocLock:YES withBlock:^(PTPDFDoc * _Nullable doc) {
```
Comment (JamesTron) [inline ios/RNTPTDocumentView.m:2101]: Lock is not required as you are not modifying or even reading the document here.

**Review by JamesTron — CHANGES_REQUESTED** [review]
One small change here.

**Review by JamesTron — APPROVED** [review]

**Pull request merged**